### PR TITLE
Jamie/fix datetime compliance reports

### PR DESCRIPTION
**Pull request by SEAjamieD**
### :nut_and_bolt: Description: What code changed, and why?
This is a continued effort to standardize time displays across the site.  This branch adds RFC2822 time format to the compliance reporting pages.  It also introduces a second time format which will be used that reads ex. `Tue, Sept 24 2019`

### :chains: Related Resources
https://github.com/chef/automate/issues/1678
https://github.com/chef/automate/issues/1679

### :+1: Definition of Done
Compliance Pages display with the new requested date format.

### :athletic_shoe: How to Build and Test the Change
1. build components/automate-ui-devproxy && start_all_services
2. Navigate to https://a2-dev.test/compliance/reports/overview
3. Click `report meta data` to view meta data --> View Report date
4. Scroll down to Chart, View Node Status Over Time Graph --> View date range under graph
5. Click on Profile Status pill to view Control Status Over Time Graph --> View date range under graph
6.  Click on `Nodes` tab, Click on any node in list to see its specific page --> View date range next to `Last Scan`
7. Click on `Scan History` in top right to view sidebar slideout --> View formatted date

### :white_check_mark: Checklist

- [ ] Tests added/updated?
- [ ] Docs added/updated?

### :camera: Screenshots, if applicable
Reports Meta Info
Reports - Meta Info

Node Status Graph
Node Status Graph

Control Status Graph
Control Status Graph

Reports - Specific Node and Sidebar
Reports - specific Node

### CODE REVIEW

File: components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-overview/overview-trend/overview-trend.component.ts [hunk 1]
```diff
@@ -153,7 +153,7 @@ export class OverviewTrendComponent implements OnChanges, OnDestroy  {
       .attr('x', this.vbWidth / 2)
       .attr('y', this.vbHeight - 3)
       .text(() => {
-        const format = '%B %e, %Y';
+        const format = '%a, %e %b %Y';
```
Comment (SEAjamieD) [inline components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-overview/overview-trend/overview-trend.component.ts]: This is a D3 time format.  Should this formatting stay here on its own or do you think I should pull this over to the datetime formatting helper?

File: components/automate-ui/src/app/helpers/datetime/datetime.ts [hunk 2]
```diff
@@ -2,4 +2,7 @@ export class DateTime {
   // formatted for use with moment.js -- https://momentjs.com/docs/#/displaying/format/
   // RFC2822 format like: Wed, 03 Jul 2019 17:08:53 UTC
   public static readonly RFC2822: string = 'ddd, DD MMM YYYY HH:mm:ss [UTC]';
+
+  // Tue, 24 Sept 2019
+  public static readonly CHEF_DATE_TIME: string = 'ddd, DD MMM YYYY';
```
Comment (SEAjamieD) [inline components/automate-ui/src/app/helpers/datetime/datetime.ts]: Open to naming suggestions.

**Comment by jonong1972** [comment]
Sweet! Looks good for me!

**Comment by SEAjamieD** [comment]
> +1 for either approaches we discussed around d3 time format.

Great idea - using moment for d3 instead.

**Pull request merged**